### PR TITLE
completed most of lab but having trouble with Album array page

### PR DESCRIPTION
**Pull request by tcoots88**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code

### CODE REVIEW

File: src/main/java/com/tcoots/songr/HomeController.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public String getRoot(Model m) {
+        return "home";
+    }
+
+    @GetMapping("/albums")
+    public String getAlbums(Model m) {
+        Album[] albums = new Album[]{new Album("Traveller", "Chris Stapleton", 14, 3784, "https://upload.wikimedia.org/wikipedia/en/d/d2/Traveller_%28Chris_Stapleton_album%29.jpg"),
+                new Album("Jekyll + Hyde", "Zac Brown Band", 16, 3962, "https://upload.wikimedia.org/wikipedia/en/d/d8/JekyllHyde.jpg"),
+                new Album("Chris Cornell", "Chris Cornell", 17, 4606, "https://upload.wikimedia.org/wikipedia/en/a/aa/Chris_Cornell_%E2%80%93_Chris_Cornell_album.png")};
+        m.addAttribute("Album-Array", albums);
```
Comment (JamesDansie) [inline src/main/java/com/tcoots/songr/HomeController.java:20]: Rename to ablumArray
